COLLABORATOR_GUIDE.md

```diff
@@ -142,6 +142,7 @@ test should *fail* before the change, and *pass* after the change.
 All pull requests that modify executable code should be subjected to
 continuous integration tests on the
 [project CI server](https://ci.nodejs.org/).
+The pull request should have a CI status indicator.
```

Comment:
Suggestion: If the CI job has the option, the pull request should have a CI status indicator.

Reply:
How about just adding "if possible"?

Reply:
OK